pyrightconfig.json

```diff
@@ -1,12 +1,5 @@
 {
   "include": ["client/valor", "api"],
-  "exclude": [
-    "client/unit-tests",
-    "utils",
-    "api/tests",
-    "examples",
-    "integration_tests",
-    "docs",
-    "migrations"
-  ]
+  "exclude": ["examples", "docs", "migrations"],
+  "reportMissingImports": "warning"
```

Comment:
the integration test files throw unusual `reportMissingImports` pyright errors without this statement. I played around with various fixes, but didn't find one that resolved both a) the tests themselves and b) the pyright errors.

marking missing imports as warnings will allow us to spot them in our files without throwing pre-commit errors where they exist